src/odo/builderImage.ts

```diff
@@ -57,26 +56,13 @@ export interface NormalizedBuilderImages {
     [builderImageName: string]: BuilderImage;
 }
 
-export class BuilderImageWrapper {
-
-    private static instance: BuilderImageWrapper;
-
-    public static get Instance(): BuilderImageWrapper {
-        if (!BuilderImageWrapper.instance) {
-            BuilderImageWrapper.instance = new BuilderImageWrapper();
-        }
-        return BuilderImageWrapper.instance;
-    }
+export class BuilderImageWrapper extends DisposableInstance {
 
     private constructor() {
```

Comment:
Is this constructor needed? Do you need to call super in it?

Reply:
I think, this `private` constructor was added in order to prevent the multiple instantiations... Not sure if this really works this way in TS/JS...
 
The `super()` call was started to be required by eslint after my changes...

Anyway, I moved the VSCommand 
```
@vsCommand('openshift.deployment.create.buildConfig')
    static async createComponent(): Promise<void> {
```
out of this module (due to the cycle dependency problem), so it's not needed to be instantiated at activation and can be rolled back to be:

```
    private static instance: BuilderImageWrapper;

    public static get Instance(): BuilderImageWrapper {
        if (!BuilderImageWrapper.instance) {
            BuilderImageWrapper.instance = new BuilderImageWrapper();
        }
        return BuilderImageWrapper.instance;
    }

    private constructor() {
        // no state
    }
```